src/main/java/org/zone/commands/structure/region/flags/entry/player/ZoneFlagEntrySetEnabledCommand.java

```diff
@@ -31,6 +31,7 @@
         return Arrays.asList(new ExactArgument("region"),
                              new ExactArgument("flag"),
                              ZONE_VALUE,
+                             new ExactArgument("player"),
```

Comment:
Can we make entry before player?

If we had it player first, it means that flags that affect both players and something else, we would need to create two commands to handle the consistanty, so my stating the player at the flag part, it means that we dont seperate all flags, just the one

Reply:
ok..